web/src/onboarding_steps.ts

```diff
@@ -57,10 +75,95 @@ function narrow_to_dm_with_welcome_bot_new_user(
     }
 }
 
+function show_onboarding_video(): void {
+    if (ONE_TIME_NOTICES_TO_DISPLAY.has("onboarding_video")) {
+        const onboarding_video_url = realm.onboarding_video_url;
+        assert(onboarding_video_url !== undefined);
+        const html_body = render_onboarding_video_modal({
+            video_src: onboarding_video_url,
+        });
+        let watch_later_clicked = false;
+        dialog_widget.launch({
+            html_heading: $t_html({defaultMessage: "Welcome to Zulip!"}),
+            html_body,
+            on_click() {
+                // Do nothing
+            },
+            html_submit_button: $t_html({defaultMessage: "Skip video — I'm familiar with Zulip"}),
+            html_exit_button: $t_html({defaultMessage: "Watch later"}),
+            close_on_submit: true,
+            id: "onboarding-video-modal",
+            close_on_overlay_click: false,
+            post_render() {
+                const $watch_later_button = $("#onboarding-video-modal .dialog_exit_button");
+                $watch_later_button.on("click", (e) => {
+                    e.preventDefault();
+                    e.stopPropagation();
+                    // Schedule a reminder message.
+                    const reminder_delay_seconds = 30;
```

Comment:
This needs an actual value. Probably 48 hours?

Reply:
Yeah, [30 was for testing](https://github.com/zulip/zulip/pull/32074#issuecomment-2716641651).

Changed it to:
> const reminder_delay_seconds = 48 * 60 * 60;